server.go

```diff
@@ -814,11 +820,11 @@ func (s *Server) Authenticate(username, password string) (*User, error) {
 	defer s.mu.Unlock()
 	u := s.users[username]
 	if u == nil {
-		return nil, fmt.Errorf("user not found")
```

Comment:
Because when it comes to security one shouldn't give away more than necessary? Return as generic a message as possible?